library/azure_ad_users.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+__version__ = "1.0.0"
```

Comment:
I would start with smaller version 0.1.0 since that code is not really in production yet :)